### ISSUE TITLE
Blender (2.79B 64-bit) crashes using the CTRL + Z shortcut.

### ISSUE THREAD

**Issue by Fabiosek** [houkama/KrxImpExp-Blender#1]
Hi.

I downloaded Blender 2.79B (64-bit) and your version of kerrax. However, when I try to use the CTRL + Z (undo) keyboard shortcut, the program crashes.  

PS. The same error occurs when you add an old version of kerax (latest 2.66 blender) to a new blender. 
PS2. The program only crashes if you load "ASC" and then use this hotkey. 

OS: Windows 10 PRO (64-bit)

**Comment by houkama** [comment]
I've tested it only for the 3ds im/exporter. Could you send me one .asc file that is causing the crash?

**Comment by Fabiosek** [comment]
> 
> 
> I've tested it only for the 3ds im/exporter. Could you send me one .asc file that is causing the crash?

Import any ASC model (e.g. body model) and press "CTRL + Z".

**File:** https://cdn.discordapp.com/attachments/467415825570332683/886585723438923807/HUM_BODY_NAKED0.ASC

**Comment by houkama** [comment]
Yes, I'm getting the same behaviour. I'll try to look into what's causing it, but cannot promise anything. It seems to work with Blender 2.66a
